content/docs/faq-versioning.md

```diff
@@ -1,64 +1,64 @@
 ---
 id: faq-versioning
-title: Versioning Policy
+title: Kebijakan Pemversian
 permalink: docs/faq-versioning.html
 layout: docs
 category: FAQ
 ---
 
-React follows [semantic versioning (semver)](https://semver.org/) principles.
+React mengikuti prinsip-prinsip [*semantic versioning* (semver)](https://semver.org/).
 
-That means that with a version number **x.y.z**:
+Yang artinya dengan nomor versi **x.y.z**:
 
-* When releasing **critical bug fixes**, we make a **patch release** by changing the **z** number (ex: 15.6.2 to 15.6.3).
-* When releasing **new features** or **non-critical fixes**, we make a **minor release** by changing the **y** number (ex: 15.6.2 to 15.7.0).
-* When releasing **breaking changes**, we make a **major release** by changing the **x** number (ex: 15.6.2 to 16.0.0).
+* Ketika merilis **perbaikan *bug* yang serius**, kita membuat sebuah **rilis *patch*** dengan mengubah nomor **z** (misal: 15.6.2 menjadi 15.6.3).
+* Ketika merilis **fitur-fitur baru** atau **perbaikan *non-critical***, kita membuat sebuah **rilis minor** dengan mengubah nomor **y** (misal: 15.6.2 menjadi 15.7.0).
+* Ketika merilis **perubahan besar**, kita membuat **rilis mayor** dengan mengubah nomor **x** (misal: 15.6.2 menjadi 16.0.0). 
 
-Major releases can also contain new features, and any release can include bug fixes.
+Rilis mayor dapat juga mengandung fitur-fitur baru, dan rilis apa pun dapat menyertakan perbaikan *bug*.
 
-Minor releases are the most common type of release.
+Rilis minor merupakan jenis rilis yang paling umum.
 
-### Breaking Changes {#breaking-changes}
+### Perubahan Besar {#breaking-changes}
```

Comment:
I don't think _Breaking Changes_ equals with _Major Changes_ especially in the context on line 23: 

> Perubahan besar tidaklah nyaman untuk semua orang

Because there are times a major version release is welcomed. so if we translated _breaking changes_ into _perubahan besar_ it's kind of misleading.

it's indeed a difficult word to translate into Bahasa Indonesia, i couldn't think of a good translation :( other than "Perubahan Tidak Kompatible"

In this case i would propose to keep _Breaking Changes_ in English

Reply:
Neither do I, actually I doubt myself while translating it because I understand the context in English but can't translate it well into Bahasa Indonesia. I will ask my friend for more comfortable translation, but I'll consider "Perubahan tidak kompatibel" as well. 

Thanks

Reply:
After asking some friends, I got some suggestions, one of them is "perubahan pematah". How about it?

Reply:
Still a weird translation to me because , looking for inspiration, i did a google translate for Malay translation of "Breaking Changes" and it got translated into _memecahkan perubahan_ so it does come close with your friend's suggestion. "pematah" can be "pemecah" or "penghancur", either these three, i think still better than "Perubahan Besar" since that would be definitely misleading.

Thank you for the effort :+1:

Reply:
After I look for France translation, they write it in a different way. Instead of "breaking changes" they use "breaks in backward compatibility". Should I adopt it as "Perubahan yang Merusak Kompatibilitas"?

Then we could translate the word "Breaking changes are inconvenient for everyone" into "Semua orang tidak menyukai perubahan-perubahan yang merusak kompatibilitas".

I think it's better, clearer, and understandable.

Reply:
it close to my original suggestion "Perubahan Tidak Kompatible" which is shorter than "Perubahan yang Merusak Kompatibilitas"

Then we translate the word "Breaking changes are inconvenient for everyone" into "Semua orang tidak menyukai perubahan-perubahan yang merusak kompatibilitas" as you suggested

Reply:
@asharimh97 After a long time of thinking, I think we're going to settle with **Perubahan Tidak Kompatibel** as a valid translation, at least for this page. Let's go with that.